src/Checkbox.jsx

```diff
@@ -14,6 +14,7 @@ class Checkbox extends Component {
     this.setState({
       isChecked: !this.state.isChecked,
     });
+    console.log('testings2', route);
```

Comment:
Please remove your console.log commands and push again!

Reply:
removed console logs

Reply:
removed the folder and icon build/favicon.ico